compiler/model/build-model.ts

```diff
@@ -188,7 +190,8 @@ function compileClassOrInterfaceDeclaration (declaration: ClassDeclaration | Int
         kind: 'request',
         name: { name, namespace: getNameSpace(declaration) },
         path: new Array<model.Property>(),
-        query: new Array<model.Property>()
+        query: new Array<model.Property>(),
+        body: { kind: 'no_body' }
```

Comment:
Having `no_body` as the default value could cause some side effects at the moment. For example, any API that we still need to define will be generated as a boolean. This could be fine tho, so we can use this "bug" to identify bad definitions.

Reply:
Indeed. I defaulted to `no_body` for now as we have many requests and responses that miss a body definition in the spec and wanted to minimize changes to the spec in this PR.

Ultimately we should remove this default. But if we decide that now is the time, I can add the missing `body` as part of this PR.

Reply:
No, let's do it in a follow-up.